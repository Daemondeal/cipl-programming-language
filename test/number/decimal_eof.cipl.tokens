Token          Lexeme         Line 
NUMBER         123            1     123
DOT            .              1    
LINE_END       \n             1    
EOF                           1    
