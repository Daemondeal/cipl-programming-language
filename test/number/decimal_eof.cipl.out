::stdout::

::stderr::
Error on line 1 at `\n`: Expect property name after '.'
